lib/commands/connect/shared.js

```diff
@@ -109,77 +110,69 @@ let request = exports.request = function(token, method, path, data){
   return promise;
 };
 
-let withUserConnections = exports.withUserConnections = function(token, appName, flags, allowNone, heroku){
+let withUserConnections = exports.withUserConnections = co.wrap(function* (token, appName, flags, allowNone, heroku){
```

Comment:
`co.wrap` returns a normal function that returns a Promise.  So I guess it really depends on what a caller to `withUserConnections` is expecting as a return value.  Since `withUserConnections` was previously returning a normal function I feel that using `co.wrap` is probably safest.